src/lib/defaultCommands.js

```diff
@@ -500,6 +505,17 @@ define(function (require) {
 				});
 
 				editor.createDropDown(caller, 'insertimage', content);
+
+				// If an image is highlighted insert details in dropdown.
+				if ($anchor.attr('width')) {
+					content.find('#width').val($anchor.attr('width'));
```

Comment:
As the default is just an empty string, could replace the if with `content.find('#width').val($anchor.attr('width') || '');`.